### PR TITLE
Youtubeme

### PR DESCRIPTION
**Pull request by Harxy**

### CODE REVIEW

File: lib/extensions/youtube-me.js [hunk 1]
```diff
@@ -1,31 +1,41 @@
...
-                var youtubeMeKeyword = "youtube me";
-                if (message.data.text.indexOf(youtubeMeKeyword) !== 0) {
+                var queueMeKeyword = "queue me";
+                if (message.data.text.indexOf(queueMeKeyword) !== 0) {
                     callback(message);
                     return;
                 }
                 dontSend = true;
-                var param = message.data.text.slice(youtubeMeKeyword.length).trim();
+                var param = message.data.text.slice(queueMeKeyword.length).trim();
```
Comment (faroligo) [inline lib/extensions/youtube-me.js]: @Harxy Isnt full youtube URLs part of this feature?

Comment (Harxy) [inline lib/extensions/youtube-me.js]: I'm taking it in steps :P

**Comment by meadsteve** [comment]
```
    Merge branch 'master' of https://github.com/Harxy/Jschat into youtubeme
```

/me looks at paul :disappointed:

**Comment by faroligo** [comment]
I was waiting for that :)

File: lib/extensions/youtube-me.js [hunk 2]
```diff
@@ -1,31 +1,43 @@
 var request = require("request");
```
Comment (meadsteve) [inline lib/extensions/youtube-me.js:1]: This file should be renamed. As it's now a generic thing queue

Comment (Harxy) [inline lib/extensions/youtube-me.js:1]: oh yeah good point

**Comment by meadsteve** [comment]
Closing in favour of #96

**Pull request closed without merging**